Makefile

```diff
@@ -14,16 +14,16 @@ all: install
 install: $(INSTALL_STAMP)
 $(INSTALL_STAMP): $(PYTHON) setup.py
 	$(VENV)/bin/pip install -U pip
-	$(VENV)/bin/pip install -Ue .
+	$(VENV)/bin/pip install --pre -Ue .
```

Comment:
Is pre necessary here?

Reply:
Yes because this code also needs jsonschema > 3